mailpoet/webpack.config.js

```diff
@@ -518,24 +518,16 @@ function requestToHandle(request) {
   /* eslint-disable-next-line consistent-return, no-useless-return */
   return;
 }
-
 const marketingOptinBlock = Object.assign({}, wpScriptConfig, {
   name: 'marketing_optin_block',
   entry: {
-    'marketing-optin-block': path.resolve(
-      process.cwd(),
-      'assets/js/src/marketing_optin_block',
-      'index.tsx',
-    ),
-    'marketing-optin-block-frontend': path.resolve(
-      process.cwd(),
-      'assets/js/src/marketing_optin_block',
-      'frontend.ts',
-    ),
+    'marketing-optin-block': '/assets/js/src/marketing_optin_block/index.tsx',
+    'marketing-optin-block-frontend':
+      '/assets/js/src/marketing_optin_block/frontend.ts',
   },
   output: {
     filename: '[name].js',
-    path: path.resolve(process.cwd(), 'assets/dist/js/marketing_optin_block'),
```

Comment:
I just want to make sure I understand the change. The `process.cwd()` was causing the nesting, correct?

Reply:
Actually it was not this causing the nesting, but was looking weird compared to the rest of the config since we are always using `__dirname` in other parts of the config.